### PR TITLE
Rename BaremetalSet to OpenStackBaremetalSet

### PR DESCRIPTION
**Pull request by dprince**

### CODE REVIEW

File: bundle.Dockerfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=osp-director-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=alpha
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.2.0
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.4.0
```
Comment (abays) [inline bundle.Dockerfile:8]: Is this the SDK version we should be using now?  It's good with me, just curious.

File: config/crd/bases/osp-director.openstack.org_openstackbaremetalsets.yaml [hunk 2]
```diff
@@ -147,9 +147,8 @@ spec:
                 secret data'
               type: string
             provisionServerName:
-              description: Name of the pre-created ProvisionServer to use. If su ProvisionServerName
-                Optional. If supplied will be used as the base Image for the baremetalset
-                instead of RhelImageURL.
+              description: ProvisionServerName Optional. If supplied will be used
+                as the base Image for the baremetalset instead of RhelImageURL.
```
Comment (abays) [inline config/crd/bases/osp-director.openstack.org_openstackbaremetalsets.yaml:151]: Nit: Here and elsewhere in comments, should `baremetalset`/`BaremetalSet` also be renamed?  I think it's pretty clear what this refers to in the context, so I could go either way.

File: config/manager/kustomization.yaml [hunk 3]
```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/openstack-k8s-operators/osp-director-operator
-  newTag: 0.0.1
+  newName: quay.io/dprince/osp-director-operator
```
Comment (abays) [inline config/manager/kustomization.yaml:7]: Should probably undo this

File: config/webhook/manifests.yaml [hunk 4]
```diff
@@ -23,15 +23,15 @@ webhooks:
...
 - clientConfig:
     caBundle: Cg==
     service:
       name: webhook-service
       namespace: system
-      path: /validate-osp-director-openstack-org-v1beta1-controlplane
+      path: /validate-osp-director-openstack-org-v1beta1-baremetalset
   failurePolicy: Fail
-  name: vcontrolplane.kb.io
+  name: vbaremetalset.kb.io
```
Comment (abays) [inline config/webhook/manifests.yaml:34]: Might need to change this to the new name

Comment (dprince) [reply config/webhook/manifests.yaml:34]: ack, this file is generate so what I missed is the changing the annotation above. Will fix...

Comment (dprince) [reply config/webhook/manifests.yaml]: @abays fixed in latest commit

File: config/manager/kustomization.yaml (hunk 3 above)
Comment (dprince) [reply config/manager/kustomization.yaml]: fixed in latest commit

**Pull request merged**